src/app/code/community/Shopgate/Cloudapi/Block/Customer/Account.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+class Shopgate_Cloudapi_Block_Customer_Account extends Mage_Adminhtml_Block_Template
+{
+    /**
+     * @return string
+     */
+    public function getToken()
+    {
+        $token = $this->getRequest()->getParam('token');
+
+        return $token;
```

Comment:
Please just directly return the getParam instead of using the $token variable.

Reply:
Also add the ```@throws Exception``` in docblock

Reply:
done